controllers/erpControllers/invoiceController.js

```diff
@@ -159,6 +160,46 @@ methods.update = async (req, res) => {
   }
 };
 
+methods.delete = async (req, res) => {
+  try {
+    const deletedInvoice = await Model.findOneAndUpdate(
+      {
+        _id: req.params.id,
+        removed: false,
+      },
+      {
+        $set: {
+          removed: true,
+        },
+      }
+    ).exec();
+
+    if (!deletedInvoice) {
+      return res.status(404).json({
+        success: false,
+        result: null,
+        message: 'Invoice not found',
+      });
+    }
+    const paymentsInvoices = await ModalPaymentInvoice.updateMany(
+      { invoiceId: deletedInvoice._id },
+      { $set: { removed: true } }
+    );
+    return res.status(200).json({
+      success: true,
+      result: {deletedInvoice, paymentsInvoices},
```

Comment:
return only  deletedInvoice
result: deletedInvoice

Reply:
I have updated the code now result only have deletedInvoice